modules/ztp-specifying-nics-in-pgt-crs-with-hub-cluster-templates.adoc

```diff
@@ -17,7 +17,14 @@ The following example shows you how to use three `ConfigMap` CRs and one `Policy
 
 [NOTE]
 ====
-When you use the `fromConfigmap` function, the `printf` variable is only available for the template resource `data` key fields. You cannot use it with `name` and `namespace` fields.
+There is a link:https://kubernetes.io/docs/concepts/configuration/configmap/#motivation[1 MiB size limit] for `ConfigMap` CRs.
```

Comment:
Doc guildelines: [When pointing to a link that exists external to Red Hat’s customer portal documentation...use the <document_heading_name> (<document_source>) format.](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute)
```suggestion
There is a link:https://kubernetes.io/docs/concepts/configuration/configmap/#motivation[1 MiB size limit] (Kubernetes documentation) for `ConfigMap` CRs.
```

Reply:
Thanks for the review @mburke5678 
I've made both suggested changes.